launch/pub_test.launch

```diff
@@ -1,6 +1,10 @@
 <launch>
     <!-- Test mavros posix sitl publishes pose at 30 Hz -->
-    <include file="$(find px4)/launch/mavros_posix_sitl.launch"/>
+    <include file="$(find px4)/launch/mavros_posix_sitl.launch">
+        <arg name="gui" value="false"/>
+        <arg name="interactive" value="false"/>
+        <arg name="verbose" value="true"/>
```

Comment:
I'm not sure if we want the verbose gazebo output?

Reply:
We do. It's useful for sitl_gazebo issues. We have this for all other CI SITL testing.